README.md

```diff
@@ -45,12 +45,13 @@ products of DESC Data Challenge 2 at the
 ["DC2 Data Product Overview"](https://confluence.slac.stanford.edu/x/oJgHDg)
 Confluence page (*DESC member only*).
 
--  **"cosmoDC2" Extragalactic Catalogs** \
+-  **"cosmoDC2" Extragalactic Catalogs and add-ons** \
    *by Andrew Benson, Andrew Hearin, Katrin Heitmann, Danila Korytov, Eve Kovacs, Patricia Larsen, Eli Rykoff et al.*
    - `cosmoDC2_v1.1.4_image`: latest cosmoDC2 catalog (used for Run 2.1+)
    - `cosmoDC2_v1.1.4_small`: 17 contiguous healpixels of `cosmoDC2_v1.1.4_image` for testing purpose
    - `cosmoDC2_v1.1.4_redmapper_v0.2.1py`: Redmapper catalog (v0.2.1) for `cosmoDC2_v1.1.4_image`.
-   - `photoz_magerr_10y_with_pdfs`: photo-z for cosmoDC2 v1.1.4 (by Sam Schmidt)
+   - `cosmoDC2_v1.1.4_image_with_photozs_v1` and `cosmoDC2_v1.1.4_small_with_photozs_v1`: containing photo-z for cosmoDC2 v1.1.4 (provided by Sam Schmidt)
+   - `cosmoDC2_v1.1.4_image_with_photoz_calib` and `cosmoDC2_v1.1.4_small_with_photoz_calib`: containing columns that indetify DESI-like QSOs, LRGs, ELGs, or a magnitude limited sample in cosmoDC2 v1.1.4 (provided by Chris Morrison)
```

Comment:
indetify -> identify

Reply:
Thanks for catching that! I think I'll need a re-approve though :slightly_smiling_face: